Author: Andreas Beckmann <anbe@debian.org>
Description: fix module build for Linux 6.4.10
 6.5: d457a0e329b0bfd3a1450e0b1a18cd2b47a25a08
 6.4.10: bbe07adbaf39c2c5a95c3ca7eb52b2119d50af7d

--- a/src/r8168_n.c
+++ b/src/r8168_n.c
@@ -84,6 +84,10 @@
 #include <asm/io.h>
 #include <asm/irq.h>
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6,4,10)
+#include <net/gso.h>
+#endif
+
 #include "r8168.h"
 #include "r8168_asf.h"
 #include "rtl_eeprom.h"
